components/switch/appearance.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Appearance
+page_title: Switch Appearance
+description: Appearance settings of the Switch for Blazor.
+slug: switch-appearance
+tags: telerik,blazor,button,switch,appearance
+published: True
+position: 35
+---
+
+# Appearance Settings
+
+You can control the appearance of the Switch button by setting the following attribute:
+
+* [Size](#size)
+* [ThumbRounded](#thumbrounded)
+* [TrackRounded](#trackrounded)
+
+
+## Size
+
+You can increase or decrease the size of the Switch by setting the `Size` attribute to a member of the `Telerik.Blazor.ThemeConstants.Switch.Size` class:
+
+| Class members | Manual declarations |
+|------------|--------|
+|`Small` |`sm`|
+|`Medium`|`md`|
+|`Large`|`lg`|
+
+>caption The built-in sizes
+
+````CSHTML
+@{
+    var fields = typeof(Telerik.Blazor.ThemeConstants.Switch.Size)
+        .GetFields(System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Static
+        | System.Reflection.BindingFlags.FlattenHierarchy)
+        .Where(field => field.IsLiteral && !field.IsInitOnly).ToList();
+
+
+    @foreach (var field in fields)
+    {
+        string size = field.GetValue(null).ToString();
+
+        <div style="float:left; margin: 20px;">
+            <TelerikSwitch @bind-Value="@isSelected" Size="@size"></TelerikSwitch>
+        </div>
+    }
+}
+
+@code{
+    private bool isSelected { get; set; }
+}
+````
+
+## ThumbRounded
```

Comment:
It might be a good idea to add a tip that for a better UI one should match the ThumbRounded and TrackRounded settings as otherwise it doesn't look good at all.

Reply:
I was thinking the same thing while looking at the rendered result, however, I thought that this would not be required as the default values match. You might be right, tho, and I will add it

Reply:
pushed that